code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm

```diff
@@ -812,3 +813,81 @@
 	R.verbs -= /mob/living/proc/shred_limb
 	R.verbs -= /mob/living/silicon/robot/proc/rest_style
 	..()
+
+
+/obj/item/weapon/robot_module/robot/booze
+	name = "BoozeHound robot module"
+	channels = list("Service" = 1)
+	pto_type = PTO_CIVILIAN
+	can_be_pushed = 0
+	sprites = list(
```

Comment:
This would be 9000% better as a greyscale sprite that gets recolored, or to simply have a full custom RGB option entirely. since it's all the same zones and bot entirely.

Reply:
yeah, we already... have code for deciding what colour a reagent should be when displayed in a transparent container

Reply:
That's something to do with the original design and I'd have no clue how to set it up.